package.json

```diff
@@ -37,6 +37,8 @@
     "react-dom": "^15.0.0 || ^16.0.0"
   },
   "dependencies": {
+    "downloadjs": "^1.4.7",
```

Comment:
I'm not particularly happy with this since it's not maintained and doesn't have much of a community. Let's use [`file-saver`](https://www.npmjs.com/package/file-saver) instead please. Let me know if you have any trouble using it